src/Tasks/NativeMethods.cs

```diff
@@ -808,6 +808,7 @@ internal static bool MakeHardLink(string newFileName, string exitingFileName, re
         // CreateSymbolicLink
         //------------------------------------------------------------------------------
         [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+        [return: MarshalAs(UnmanagedType.I1)]
```

Comment:
Native `CreateSymbolicLink` returns `BOOLEAN` which is a typedef for `BYTE`. Therefore, according to https://blogs.msdn.microsoft.com/jaredpar/2008/10/14/pinvoke-and-bool-or-should-i-say-bool/, `MarshalAs(UnmanagedType.I1)` should be used here.

Reply:
Nit, optional: Can you update the other pinvokes that return bool, at least in this file?

Reply:
As far as I can see, this was only a problem for `CreateSymbolicLink`, because it returns `BOOLEAN` (which is 1 byte only). Other pinvokes there, return `BOOL`  which is 4-byte type and that is correctly translated into bool.